2024-10-21 18:19:00,703 : WARN  : comm-pool-thread-4 :  : Node : XML Writer : 3:14 : The selected XML column 'XML' is not part of the input
2024-10-21 18:20:11,468 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:20:15,027 : WARN  : comm-pool-thread-4 :  : Node : XML Writer : 3:14 : The selected XML column 'XML' is not part of the input
2024-10-21 18:20:23,830 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:21:11,192 : WARN  : main :  : WrappedNodeDialog : XML Writer : 3:14 : failed to apply settings: The file name pattern is not valid.
org.knime.core.node.InvalidSettingsException: The file name pattern is not valid.
	at org.knime.filehandling.core.node.table.writer.AbstractMultiTableWriterNodeConfig$1.validateSettingsForModel(AbstractMultiTableWriterNodeConfig.java:169)
	at org.knime.core.node.defaultnodesettings.SettingsModel.validateSettings(SettingsModel.java:337)
	at org.knime.filehandling.core.node.table.writer.DefaultMultiTableWriterNodeConfigSerializer.validate(DefaultMultiTableWriterNodeConfigSerializer.java:142)
	at org.knime.filehandling.core.node.table.writer.DefaultMultiTableWriterNodeConfigSerializer.validate(DefaultMultiTableWriterNodeConfigSerializer.java:1)
	at org.knime.filehandling.core.node.table.writer.AbstractMultiTableWriterNodeConfig.validateSettingsForModel(AbstractMultiTableWriterNodeConfig.java:270)
	at org.knime.filehandling.core.node.table.writer.AbstractMultiTableWriterNodeModel.validateSettings(AbstractMultiTableWriterNodeModel.java:453)
	at org.knime.core.node.Node.validateSettings(Node.java:711)
	at org.knime.core.node.Node.validateModelSettings(Node.java:683)
	at org.knime.core.node.workflow.NativeNodeContainer.performValidateSettings(NativeNodeContainer.java:1193)
	at org.knime.core.node.workflow.SingleNodeContainer.validateSettings(SingleNodeContainer.java:1106)
	at org.knime.core.node.workflow.WorkflowManager.loadNodeSettings(WorkflowManager.java:2225)
	at org.knime.core.node.workflow.NodeContainer.applySettingsFromDialog(NodeContainer.java:1141)
	at org.knime.core.ui.wrapper.NodeContainerWrapper.applySettingsFromDialog(NodeContainerWrapper.java:264)
	at org.knime.workbench.ui.wrapper.WrappedNodeDialog.callApplySettingsFromDialog(WrappedNodeDialog.java:671)
	at org.knime.workbench.ui.wrapper.WrappedNodeDialog.doApply(WrappedNodeDialog.java:643)
	at org.knime.workbench.ui.wrapper.WrappedNodeDialog$6.widgetSelected(WrappedNodeDialog.java:509)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:252)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4274)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4072)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3660)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:823)
	at org.eclipse.jface.window.Window.open(Window.java:799)
	at org.knime.workbench.ui.wrapper.WrappedNodeDialog.open(WrappedNodeDialog.java:198)
	at org.knime.workbench.editor2.editparts.NodeContainerEditPart.openDialog(NodeContainerEditPart.java:961)
	at org.knime.workbench.editor2.editparts.NodeContainerEditPart.openNodeDialog(NodeContainerEditPart.java:868)
	at org.knime.ui.java.api.NodeAPI.openNodeDialog(NodeAPI.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.knime.ui.java.api.DesktopAPI.invokeMethod(DesktopAPI.java:174)
	at org.knime.ui.java.api.DesktopAPI.lambda$1(DesktopAPI.java:145)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:132)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4047)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3663)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1151)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1042)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:152)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:639)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:546)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.knime.product.rcp.KNIMEApplication.start(KNIMEApplication.java:211)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:143)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:109)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:439)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:271)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)
2024-10-21 18:21:15,525 : WARN  : main :  : Node : XML Writer : 3:14 : Please specify a folder
2024-10-21 18:21:16,094 : WARN  : main :  : Node : XML Writer : 3:14 : Please specify a folder
2024-10-21 18:21:16,824 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:21:16,825 : WARN  : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Writer : 3:14 : Please specify a folder
2024-10-21 18:21:23,550 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:21:50,809 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:21:51,459 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:21:51,870 : ERROR : KNIME-Worker-1-XML Row Combiner 3:15 :  : Node : XML Row Combiner : 3:15 : Execute failed: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
java.lang.IllegalStateException: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:233)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 11
	at org.knime.core.data.container.BlobSupportDataRow.getCell(BlobSupportDataRow.java:144)
	at org.knime.xml.node.rcombine.XMLRowCombinerNodeModel.execute(XMLRowCombinerNodeModel.java:219)
	... 14 more
2024-10-21 18:22:14,835 : WARN  : main :  : Node : Column to XML : 3:13 : Cannot create column "CustomerID" since it is already in the input.
2024-10-21 18:22:15,337 : WARN  : main :  : Node : Column to XML : 3:13 : Cannot create column "CustomerID" since it is already in the input.
2024-10-21 18:22:35,108 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:36,170 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:36,755 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:37,249 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:37,479 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:37,850 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:51,024 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:51,865 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:22:52,194 : ERROR : KNIME-Worker-1-Column to XML 3:13 :  : Node : Column to XML : 3:13 : Execute failed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
java.lang.IllegalStateException: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:405)
	at org.knime.core.data.container.SingleCellFactory.getCell(SingleCellFactory.java:128)
	at org.knime.core.data.container.SingleCellFactory.getCells(SingleCellFactory.java:105)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewCellsForRow(RearrangeColumnsTable.java:565)
	at org.knime.core.data.container.RearrangeColumnsTable.calcNewColsSynchronously(RearrangeColumnsTable.java:484)
	at org.knime.core.data.container.RearrangeColumnsTable.create(RearrangeColumnsTable.java:412)
	at org.knime.core.data.container.BufferedTableBackend.rearrange(BufferedTableBackend.java:212)
	at org.knime.core.node.ExecutionContext.createColumnRearrangeTable(ExecutionContext.java:467)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel.execute(ColumnToXMLNodeModel.java:194)
	at org.knime.core.node.NodeModel.execute(NodeModel.java:812)
	at org.knime.core.node.NodeModel.executeModel(NodeModel.java:588)
	at org.knime.core.node.Node.invokeFullyNodeModelExecute(Node.java:1286)
	at org.knime.core.node.Node.execute(Node.java:1049)
	at org.knime.core.node.workflow.NativeNodeContainer.performExecuteNode(NativeNodeContainer.java:594)
	at org.knime.core.node.exec.LocalNodeExecutionJob.mainExecute(LocalNodeExecutionJob.java:98)
	at org.knime.core.node.workflow.NodeExecutionJob.internalRun(NodeExecutionJob.java:198)
	at org.knime.core.node.workflow.NodeExecutionJob.run(NodeExecutionJob.java:117)
	at org.knime.core.util.ThreadUtils$RunnableWithContextImpl.runWithContext(ThreadUtils.java:367)
	at org.knime.core.util.ThreadUtils$RunnableWithContext.run(ThreadUtils.java:221)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at org.knime.core.util.ThreadPool$MyFuture.run(ThreadPool.java:123)
	at org.knime.core.util.ThreadPool$Worker.run(ThreadPool.java:246)
Caused by: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 13; Attribute name "Card" associated with an element type "Credit" must be followed by the ' = ' character.
	at java.xml/com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(Unknown Source)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(Unknown Source)
	at org.knime.core.data.xml.io.XMLDOMCellReader.readXML(XMLDOMCellReader.java:169)
	at org.knime.core.data.xml.XMLCellContent.parse(XMLCellContent.java:274)
	at org.knime.core.data.xml.XMLCellContent.<init>(XMLCellContent.java:105)
	at org.knime.core.data.xml.XMLCellFactory.create(XMLCellFactory.java:139)
	at org.knime.xml.node.columntoxml.ColumnToXMLNodeModel$ColumnToXMLCellFactory.getCell(ColumnToXMLNodeModel.java:403)
	... 22 more
2024-10-21 18:23:59,421 : ERROR : main :  : KnimeBrowserView :  :  : %cerror%c Error making RPC call 
      background: #c0392b;
      border-radius: 0.5em;
      color: white;
      font-weight: bold;
      padding: 2px 0.5em;
      Error: No such node ID: 3:14 valid [3:1, 3:2, 3:3, 3:4, 3:5, 3:6, 3:7, 3:8, 3:9, 3:10, 3:11, 3:13, 3:15] (source: http://org.knime.ui.java/assets/index-PyhZB-uQ.js; line: 64)
2024-10-21 18:23:59,421 : ERROR : main :  : KnimeBrowserView :  :  : Uncaught (in promise) Error: No such node ID: 3:14 valid [3:1, 3:2, 3:3, 3:4, 3:5, 3:6, 3:7, 3:8, 3:9, 3:10, 3:11, 3:13, 3:15] (source: http://org.knime.ui.java/assets/index-PyhZB-uQ.js; line: 22)
2024-10-21 18:25:00,425 : WARN  : comm-pool-thread-5 :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:25:11,248 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:25:11,248 : WARN  : main :  : Node : GroupBy : 3:10 : All columns selected as group by column
2024-10-21 18:25:11,698 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:25:11,699 : WARN  : main :  : Node : GroupBy : 3:10 : All columns selected as group by column
2024-10-21 18:25:12,808 : WARN  : KNIME-Worker-2-Column Filter 3:11 :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:25:12,809 : WARN  : KNIME-Worker-2-Column Filter 3:11 :  : Node : GroupBy : 3:10 : All columns selected as group by column
2024-10-21 18:25:46,697 : WARN  : main :  : Node : GroupBy : 3:10 : No grouping column included. Aggregate complete table.
2024-10-21 18:25:47,340 : WARN  : main :  : Node : GroupBy : 3:10 : No grouping column included. Aggregate complete table.
2024-10-21 18:26:08,455 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:26:08,978 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:26:18,214 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:26:21,858 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:26:22,224 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:27:00,422 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:27:01,051 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:27:07,983 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:27:09,496 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:27:24,160 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:27:24,667 : WARN  : main :  : Node : GroupBy : 3:10 : No aggregation column defined
2024-10-21 18:31:29,749 : WARN  : WebSocket@968378519-2198 :  : Node : Unpivot : 3:17 : This node was auto-configured. No columns have been specified in the configuration. Now all suitable columns in 'value' and 'retained' column lists are used.
2024-10-21 18:32:36,556 : ERROR : main :  : KnimeBrowserView :  :  : %cerror%c Error making RPC call 
      background: #c0392b;
      border-radius: 0.5em;
      color: white;
      font-weight: bold;
      padding: 2px 0.5em;
      Error: No such node ID: 3:17 valid [3:1, 3:2, 3:3, 3:4, 3:5, 3:6, 3:7, 3:8, 3:9, 3:10, 3:11, 3:13, 3:15, 3:16] (source: http://org.knime.ui.java/assets/index-PyhZB-uQ.js; line: 64)
2024-10-21 18:32:36,556 : ERROR : main :  : KnimeBrowserView :  :  : Uncaught (in promise) Error: No such node ID: 3:17 valid [3:1, 3:2, 3:3, 3:4, 3:5, 3:6, 3:7, 3:8, 3:9, 3:10, 3:11, 3:13, 3:15, 3:16] (source: http://org.knime.ui.java/assets/index-PyhZB-uQ.js; line: 22)
2024-10-21 18:37:34,001 : WARN  : AWT-EventQueue-0 :  : CSVFormatAutoDetectionSwingWorker : CSV Reader : 3:1 : org.knime.core.node.InvalidSettingsException: The specified file invoices.csv does not exist.
java.util.concurrent.ExecutionException: org.knime.core.node.InvalidSettingsException: The specified file invoices.csv does not exist.
	at java.base/java.util.concurrent.FutureTask.report(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.get(Unknown Source)
	at java.desktop/javax.swing.SwingWorker.get(Unknown Source)
	at org.knime.base.node.io.filehandling.csv.reader.CSVFormatAutoDetectionSwingWorker.doneWithContext(CSVFormatAutoDetectionSwingWorker.java:134)
	at org.knime.core.util.SwingWorkerWithContext.done(SwingWorkerWithContext.java:163)
	at java.desktop/javax.swing.SwingWorker$5.run(Unknown Source)
	at java.desktop/javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.run(Unknown Source)
	at java.desktop/sun.swing.AccumulativeRunnable.run(Unknown Source)
	at java.desktop/javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.actionPerformed(Unknown Source)
	at java.desktop/javax.swing.Timer.fireActionPerformed(Unknown Source)
	at java.desktop/javax.swing.Timer$DoPostEvent.run(Unknown Source)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
Caused by: org.knime.core.node.InvalidSettingsException: The specified file invoices.csv does not exist.
	at org.knime.core.node.util.CheckUtils.checkSetting(CheckUtils.java:273)
	at org.knime.core.node.util.CheckUtils.checkSetting(CheckUtils.java:223)
	at org.knime.filehandling.core.defaultnodesettings.filechooser.AbstractFileChooserPathAccessor.getRootPath(AbstractFileChooserPathAccessor.java:318)
	at org.knime.filehandling.core.defaultnodesettings.filechooser.AbstractFileChooserPathAccessor.getFSPaths(AbstractFileChooserPathAccessor.java:225)
	at org.knime.filehandling.core.defaultnodesettings.filechooser.reader.ReadPathAccessor.getPaths(ReadPathAccessor.java:96)
	at org.knime.base.node.io.filehandling.csv.reader.CSVFormatAutoDetectionSwingWorker.doInBackgroundWithContext(CSVFormatAutoDetectionSwingWorker.java:99)
	at org.knime.base.node.io.filehandling.csv.reader.CSVFormatAutoDetectionSwingWorker.doInBackgroundWithContext(CSVFormatAutoDetectionSwingWorker.java:1)
	at org.knime.core.util.SwingWorkerWithContext.doInBackground(SwingWorkerWithContext.java:106)
	at java.desktop/javax.swing.SwingWorker$1.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.desktop/javax.swing.SwingWorker.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
